packages/core-data/src/actions.js

```diff
@@ -888,6 +888,13 @@ export function receiveUserPermission( key, isAllowed ) {
 	};
 }
 
+export function receiveUserPermissions( permissions ) {
```

Comment:
Let's add some docs:

```suggestion
/**
 * Returns an action object used in signalling that all permissions of the
 * current user have been received.
 *
 * @ignore
 * 
 * @param {Object} permissions User permissions
 *
 * @return {Object} Action object.
 */
export function receiveUserPermissions( permissions ) {
```